### ISSUE TITLE
Better interop with s3fs-fuse

### ISSUE THREAD

**Issue by nzjrs** [bloomreach/s4cmd#19]
AFAICT through testing, https://github.com/s3fs-fuse/s3fs-fuse is the only currently working and developed fuse filesystem for s3. In order to better interoperate with then I suggest setting the following additional metadata in a way compatible with that tool

x-amz-meta-gid
x-amz-meta-mtime
x-amz-meta-uid
x-amz-meta-mode / x-amz-meta-permissions

**Comment by chouhanyang** [comment]
Good point. Previously, we tried to be compatible s3cmd, which has its own implementations. I'll look into s3fuse's implementation in this case.

**Comment by a7medo778** [comment]
Any update on this

**Comment by geekpete** [comment]
Would be nice
